sonar/modules/documents/mappings/v7/documents/document-v1.0.0.json

```diff
@@ -388,6 +388,10 @@
                 "fields": {
                   "raw": {
                     "type": "keyword"
+                  },
+                  "suggest": {
+                    "type": "completion",
+                    "analyzer": "default"
```

Comment:
Didn't know this `type` and `analyzer`. I need to learn about them !

Reply:
It's the built-in type for having completion in ES: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters.html#completion-suggester